pkg/ovsdb/handler.go

```diff
@@ -255,14 +255,22 @@ func (ch *Handler) MonitorCondChange(ctx context.Context, params []interface{})
 		if !ok {
 			ch.log.Info("MonitorCondChange there is no monitor", "dbname", monitorData.dataBaseName)
 		}
+		schema, ok := ch.db.GetSchemas()[dbName]
+		if !ok {
+			return nil, fmt.Errorf("there is no schema for %s", dbName)
+		}
 		for tableName, mcrArray := range mcrs {
 			key := common.NewTableKey(dbName, tableName)
 			_, ok := monitor.key2Updaters[key]
 			if !ok {
 				ch.log.V(6).Info("MonitorCondChange", "table", tableName, "mcr", mcrArray)
 				var updaters []updater
+				tableSchema, ok := schema.Tables[tableName]
```

Comment:
You can use the helper function LookupTable() rather than map lookup, it provides am error string